packages/core/src/sass/_globals.scss

```diff
@@ -585,12 +587,14 @@ $AU-border-radius: 3px !default;
 		$ratio: 1 / $ratio;
 	}
 
-	$ratio: $ratio;
-
 	@if( $ratio < 4.5 and $silent == false ) {
 		@warn "Houston, we have a problem; contrast ratio of #{ $forground } on #{ $background } is bad, just #{ $ratio }";
 	}
 
+	@if $rounded == true {
+		$ratio: round( $ratio * 10 ) / 10;
+	}
+
```

Comment:
do an `@else` he to be more explicit

Reply:
Added else

Reply:
We just want a way to `round` the ratio for displaying the color contrast boxes without a long integer. I don't think it was related to the `@if` above.